vertical-pod-autoscaler/pkg/updater/main.go

```diff
@@ -45,6 +46,12 @@ var (
 	evictionToleranceFraction = flag.Float64("eviction-tolerance", 0.5,
 		`Fraction of replica count that can be evicted for update, if more than one pod can be evicted.`)
 
+	evictionRateLimit = flag.Float64("eviction-rate-limit", -1, `
+		Number of pods that can be evicted per seconds.`)
```

Comment:
Please add info that 0 and -1 turn rate limiting off. Also, I think the ` should be on the next line.

Reply:
👍 Done